patches/eslint-plugin-import+2.26.0.patch

```diff
@@ -0,0 +1,265 @@
+diff --git a/node_modules/eslint-plugin-import/config/recommended-esm.js b/node_modules/eslint-plugin-import/config/recommended-esm.js
```

Comment:
will be able to remove when https://github.com/import-js/eslint-plugin-import/pull/2701 is merged